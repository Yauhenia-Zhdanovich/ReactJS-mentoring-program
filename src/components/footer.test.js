import React from 'react';
import renderer from 'react-test-renderer';

import Footer from './footer.jsx';

describe('Footer snap', () => {
  it('renders Footer snapshot', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
