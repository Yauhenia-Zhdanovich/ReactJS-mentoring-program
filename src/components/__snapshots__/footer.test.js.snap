// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snap renders Footer snapshot 1`] = `
<footer
  className="sc-bdnylx fwjQbp"
>
  <p>
    netflix roulette
  </p>
</footer>
`;
